app/livechat/server/api/v1/visitor.ts

```diff
@@ -37,13 +37,15 @@ API.v1.addRoute('livechat/visitor', {
 		}
 
 		guest.connectionData = normalizeHttpHeaderData(this.request.headers);
-		const visitorId = Livechat.registerGuest(guest);
+		const visitorId = Livechat.registerGuest(guest as any);
```

Comment:
Is this `any` really necessary? If so, pls use `unknown` instead

Reply:
yes... is necessary! The Livechat.registerGuest function is in a .js file with a destructured object as guest... I can't pass this as unknown

Reply:
Then can you add a TODO there saying that we need to convert the js file and the name of the file. I can create a task for it, so we don't forget